CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # git-dit ChangeLog
 
+## v0.4.0 (2017-09-15)
+
+### Binary
+
+Changes:
+ * The message formatting for the "show" subcommand was improved.
+ * In filter-specs for the "list" subcommand, a '=' is used for separating key
+   and value when filtering for an exact value, rather than a ':'.
+
+Added features:
+ * The "create-message", "new", "reply" and "tag" now accept the "--author" and
+   "--date" options. Analogue to the options of "git commit" with the same name,
+   the new options alter the author and authoring date of the resulting message.
+ * New "reporter-name" and "reporter-email" filter keys for issue filtering
+   using the "list" subcommand. As the name suggests, the keys may be used for
+   filtering issues based on the reporter's name or email.
```

Comment:
Unclear what a "reporter" is in this context: Is it the author of the issue (as in the git-commit author) or a dit-trailer `Dit-author:` or something like that. Please clearify.

Reply:
This information does not belong in the Changelog but in the manpage/user documentaton.

Because of the "list" subcommand alone, we should introduce subcommand-specific manpages imo, btw.

Reply:
I'm against subcommand specific manpages,... having one huge manpage is better for user experience because you can simply search through one manpage and you know that if you did not find what you were looking for, it is not there.

At least with the current amount of features, multiple manpages make no sense IMO. Of course we could reconsider this if we hit a 1.0 for example, and there are much more flags and subcommands around...